src/store/ordinoSlice.ts

```diff
@@ -83,6 +83,14 @@ export interface IWorkbench {
    */
   selection: IRow['_visyn_id'][];
 
+  formatting?: {
```

Comment:
Please add a comment that describes where/for what the formatting is currently used?

Reply:
I saw now in https://github.com/datavisyn/reprovisyn/pull/310/files#diff-20822a6e7462d4b6c46d91c29bdb47cbae71b25d8a2acdfc84547db4fb63d077R77 that this should be of type `IEntityFormatting` but we need to wait for @oltionchampari refactoring, right?